src/hooks/useDrawPath.tsx

```diff
@@ -1,21 +1,30 @@
-import { useState } from 'react';
+import { useMemo, useState } from 'react';
 
+import { SeriesPoint } from '..';
+import { Polygon } from '../components/Annotations/Polygon';
 import { Polyline } from '../components/Annotations/Polyline';
 
-import { DualAxisOptions, PathOptions } from './types';
+import { ControllerHookOptions, DualAxisOptions, PathOptions } from './types';
 import { useStartMoveEnd } from './useStartMoveEnd';
 
-export interface UseDrawPathOptions extends DualAxisOptions, PathOptions {
-  onDraw?: (path: { data: Record<string, number>[] }) => void;
+export interface UseDrawPathOptions
+  extends ControllerHookOptions,
+    DualAxisOptions,
+    PathOptions {
+  close?: boolean;
+  onDraw?: (points: SeriesPoint[]) => void;
+  onEnd?: (points: SeriesPoint[]) => void;
```

Comment:
this is inconsistent with useDrawRectangle, which calls `onDraw` at the end, not on each move.

Reply:
> You can rename `onDraw` -> `onEnd`. `onDraw` is semantically too close to `onDrawing`

it was 
onDraw for the onEnd 
and onDrawing for onMove

Reply:
I'm still in favor of `onEnd`, but indeed it would have to be used consistently in all hooks.

I don't like `onDraw` because I understand it as "when the hook draws something, call this callback", but the hook in itself actually stops drawing on end

Reply:
i also agree with @stropitek

Reply:
it's good to me if we also rename the callback in the other hook.